src/woo_publications/publications/tests/factories.py

```diff
@@ -18,3 +20,17 @@ class DocumentFactory(factory.django.DjangoModelFactory):
 
     class Meta:  # type: ignore
         model = Document
+
+    class Params:
+        document_service_configured = factory.Trait(
+            document_service=factory.SubFactory(
+                ServiceFactory,
+                api_root="https://example.com/",
+                api_type=APITypes.drc,
+                oas="https://example.com/api/v1/oas",
+                header_key="Authorization",
+                header_value="Token 0cbccf9e-f9cd-4f9c-9516-7481e79989df",
+                auth_type=AuthTypes.api_key,
```

Comment:
That's not an accurate reflection, since the Documents API use ZGW auth rather than token/API key auth.

Also, for integration with VCR etc. we can point to the Open Zaak instance in our `docker-compose.yml` instead - that makes testing a lot simpler!

Reply:
I think I configured this correctly but please double check for me.